content/posts/scientific-python/translations/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Setting up and managing translations for Scientific Python projects"
```

Comment:
The title is quite long, perhaps consider simply "Translations for Scientific Python projects" or "Managing translations for Scientific Python projects".